beacon-chain/rpc/service.go

```diff
@@ -103,11 +108,9 @@ func NewRPCService(ctx context.Context, cfg *Config) *Service {
 // Start the gRPC server.
 func (s *Service) Start() {
 	log.Info("Starting service")
-
 	lis, err := net.Listen("tcp", fmt.Sprintf(":%s", s.port))
 	if err != nil {
-		log.Errorf("Could not listen to port :%s: %v", s.port, err)
-		return
+		log.Fatalf("Could not listen to port in Start() :%s: %v", s.port, err)
```

Comment:
why `Fatal` here, this is going to kill the entire thing

Reply:
+1, why change the severity of this message?

Reply:
Updated to log.Errorf